api/v1/openapi.yaml

```diff
@@ -555,6 +555,7 @@ paths:
       - policy
       parameters:
       - "$ref": "#/parameters/policy-rules"
+      - "$ref": "#/parameters/policy-validate"
```

Comment:
It doesn't make sense to me to have an API change that modifies validation behaviour. Validation protects the internals of the agent, so should be applied unconditionally server-side. In general, delegating the option to validate to clients is not great form since a client that is acting poorly could then endanger the reliability of the server.

Reply:
This is not completely true, the validation actually happens unconditionally server-side anyway.  With the current upstream code it only happens twice (once client-side, once server-side). The API change here is only added here to be able to reuse PUT API call.

Reply:
Maybe to take a step back, could you describe why the client needs to make different API calls to change what the client does? Why wouldn't this just be a CLI flag?

Reply:
you are right, I don't..